lib/inbox/pages/inbox_page.dart

```diff
@@ -48,26 +49,46 @@ class _InboxPageState extends State<InboxPage> {
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
+    final AppLocalizations l10n = AppLocalizations.of(context)!;
 
     return Scaffold(
       appBar: AppBar(
         toolbarHeight: 80.0,
         centerTitle: false,
-        title: AutoSizeText(AppLocalizations.of(context)!.inbox, style: theme.textTheme.titleLarge),
+        title: AutoSizeText(l10n.inbox, style: theme.textTheme.titleLarge),
         actions: [
           IconButton(
             icon: Icon(
               Icons.checklist,
-              semanticLabel: AppLocalizations.of(context)!.readAll,
+              semanticLabel: l10n.readAll,
             ),
-            onPressed: () {
-              context.read<InboxBloc>().add(MarkAllAsReadEvent());
+            onPressed: () async {
+              bool result = false;
+              await showThunderDialog<bool>(
+                context: context,
+                title: l10n.confirmMarkAllAsReadTitle,
```

Comment:
I think the title/headline "Mark All As Read?" should suffice since the dialog itself acts as the confirmation modal (although I did notice that the log out modal also has "Confirm", so maybe we can change the "Confirm Log Out" to just "Log Out?" to make it consistent)

It also shortens the title and makes it a single line which looks better imo! (https://m3.material.io/components/dialogs/guidelines also mostly shows single lined headline)

Reply:
Sure thing, I updated both.

| ![image](https://github.com/thunder-app/thunder/assets/7417301/cb586101-b2ed-4ad5-9fa0-58ec889c86fe) |
| - |